pkg/modprobe/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM linuxkit/alpine:87a0cd10449d72f374f950004467737dbf440630 as mirror
+
+RUN mkdir -p /out/etc/apk && cp -r /etc/apk/* /out/etc/apk/
+RUN apk add --no-cache --initdb -p /out \
+    musl \
+    && true
+RUN rm -rf /out/etc/apk /out/lib/apk /out/var/cache
+RUN mkdir /out/bin && mkdir /out/sbin && cp /bin/busybox /out/bin/busybox
+
+FROM scratch
+ENTRYPOINT []
+CMD []
+WORKDIR /
+COPY --from=mirror /out/ /
+RUN ["/bin/busybox", "ln", "-s", "/bin/busybox", "/sbin/modprobe"]
```

Comment:
This is simply to keep the symlink to /bin/busybox in place, rather than letting the `COPY` place the /bin/busybox binary in /sbin as /sbin/modprobe

Reply:
Is there a reason why you use a different format of the `RUN` command here compare to above? I prefer consistency

Reply:
It avoids an unnecessary shell (the string version is mutated into `sh -c 'foo'`). Not sure why that is an issue here though.

Reply:
Thanks. I am now copying the `/sbin/modprobe` symlink properly with `cp -P`, rather than creating it in the final stage.